src/Security/Authentication/OpenIdConnect/src/OpenIdConnectHandler.cs

```diff
@@ -36,10 +36,24 @@ public class OpenIdConnectHandler : RemoteAuthenticationHandler<OpenIdConnectOpt
 
         private OpenIdConnectConfiguration _configuration;
 
+        /// <summary>
+        /// Gets the <see cref="HttpClient"/> used to communicate with the remote identity provider.
+        /// </summary>
         protected HttpClient Backchannel => Options.Backchannel;
 
+        /// <summary>
+        /// Gets the <see cref="System.Text.Encodings.Web.HtmlEncoder"/>.
```

Comment:
Namespace?

Reply:
It resolves to the property otherwise. We want to the point to the type. Btw, the property is unused from what I could tell.